cityofaustin/atd-moped
# Replace "Current status" filter in advanced search with project badge values #7831

## Associated issues

Closes https://github.com/cityofaustin/atd-data-tech/issues/7831

## Testing
Moped-Test:
https://moped-test.austinmobility.io/moped/

**Steps to test:**

1. Create a new project, or use an existing project and make sure the status badge is visible.
2. Search "current status" by the current status (potential, complete, etc).


https://user-images.githubusercontent.com/5282430/148068376-476f95d9-aab5-4f50-8108-8642ce052bed.mp4



---
#### Ship list
